larndsim/consts/detector.py

```diff
@@ -74,8 +87,52 @@
 MODULE_TO_TPCS = {}
 TPC_TO_MODULE = {}
 
-ELECTRON_MOBILITY_PARAMS = 551.6, 7158.3, 4440.43, 4.29, 43.63, 0.2053
-
+###################
+# LArPix FEE
+###################
+#: Discrimination threshold in e-
+DISCRIMINATION_THRESHOLD = 7e3 # e-
```

Comment:
I suggest the changing default to 5e3

Reply:
This will be overwrite by the yaml. I left it there as we've been running it with 7e3 or so. I think we should have a separate discussion about the threshold, but I don't have strong opinion about the default values.